### DISCUSSION TITLE
4700G AMD Radeon RX Vega10 Unknown Prototype.

### DISCUSSION THREAD

**Discussion by CraigHazan** [ChefKissInc/NootedRed#136] [General]
After a successful first build with the 4600G, I found a good price for the 4700G on eBay, expecting it later this week. Interested to see how 8 Radeon Vega iGPU cores beating at 2.1 MHz looks in Sonoma, specifically No Mans Sky.

**Comment by CraigHazan** [comment]
Successfully upgraded my APU to a Ryzen 7 4700G. Oh.. I also changed my Device Property for the VGA Compatible Controller, now just Renoir 8GB. GeekBench 6 However sees a **AMD Radeon RX Vega10 Unknown Prototype**. CaseySJ noticed that 'you're injecting AAPL,ig-platform-id = 0x3EA50009. That seems to set the framebuffer.'. That came from exporting the .plist from Hackintool and copying it to my EFI. 


![Screenshot 2023-07-26 at 5 55 55 PM](https://github.com/NootInc/NootedRed/assets/20331487/bd7100fb-3790-4e4c-864f-2fa08c5a6481)

**Reply by truongvantuan** [reply]
Omg, i have a 4700ge, the 8G APU vram show in your is default of get by patch?

**Reply by CraigHazan** [reply]
You can change the amount of VRAM in BIOS. ie I have 2 x 8 GB installed I can choose up to 8 GB VRAM in BIOS. 1 x 8 GB installed I can choose up to 4 GB VRAM.

**Reply by truongvantuan** [reply]
> You can change the amount of VRAM in BIOS. ie I have 2 x 8 GB installed I can choose up to 8 GB VRAM in BIOS. 1 x 8 GB installed I can choose up to 4 GB VRAM.

My bios just can set to 512MB. I was hope that you can set VRAM to 8GB just by config.plist opencore file.

**Comment by VisualEhrmanntraut** [comment]
> you're injecting AAPL,ig-platform-id = 0x3EA50009. That seems to set the framebuffer.

No I'm not.

**Reply by CraigHazan** [reply]
No one said ChefKissinc is injecting the frame buffer. I changed the Device Property, which implies I injected the Framebuffer, which 'I' did when I copied the the Device Property to the EFI.

**Reply by VisualEhrmanntraut** [reply]
Grammatically, I parsed that you meant NRed added this property. Anyway, that framebuffer ID is for Intel iGPUs; it does nothing here, I'd even bet it could be harmful. Hackintool was not made to be used on AMD systems anyway, it's pretty focused on Intel and Clover

**Reply by CraigHazan** [reply]
could you possibly let me know what I should be calling it?.

**Reply by CraigHazan** [reply]
Just to add. Activity Monitor and Geekbench 6 label the APU graphics AMD Radeon Vega10 Unknown Prototype. does that get fixed with a framebuffer?

**Reply by VisualEhrmanntraut** [reply]
This name comes from the OpenGL library AMD made, you cannot change it with a property; it is based on the device type. Are you using an old version of NRed? Because, on Renoir it should've showed Vega20, not Vega10.

**Reply by CraigHazan** [reply]
No, latest NRed from a couple of days ago. Yes the Framebuffer was for Intel Coffee Lake, I see what's happening now, confused with my 'ID's. So I need to change the Platform ID, but I can't remember what it was before, oops.

**Reply by VisualEhrmanntraut** [reply]
No, you don't need to change the ID. You need to remove these properties

**Reply by CraigHazan** [reply]
Thank you, I appreciate your time @ChefKissInc, I know you're busy.

**Reply by CraigHazan** [reply]
I removed the Device Property, but it still says Vega10 Unknown Prototype.
![Screenshot 2023-07-27 at 8 50 36 AM](https://github.com/NootInc/NootedRed/assets/20331487/632dd4e1-2951-49f1-b8e5-a3cfe813b4ab)

**Reply by VisualEhrmanntraut** [reply]
> This name comes from the OpenGL library AMD made, you cannot change it with a property; it is based on the device type.

**Reply by CraigHazan** [reply]
It still looks beautiful on a 4700G, just tiny details you notice and the extra oomph over the 4600G. Incredible work @ChefKissInc and the team.

**Comment by CraigHazan** [comment]
Seems to fixed on a Gigabyte B450M DS3H WIFI.

![Screenshot 2023-07-31 at 5 24 21 AM](https://github.com/NootInc/NootedRed/assets/20331487/86d56706-266b-4afe-bac1-eec63d956613)

**Comment by paranoiagu** [comment]
also see "AMD Radeon RX Vega10 Unknown Prototype" on my Ryzen R7 4800H

**Reply by VisualEhrmanntraut** [reply]
Not a bug.
